griptape/artifacts/json_artifact.py

```diff
@@ -12,10 +12,20 @@
 
 @define
 class JsonArtifact(BaseArtifact):
-    value: Json = field(converter=lambda v: json.loads(json.dumps(v)), metadata={"serializable": True})
+    """Stores JSON data.
+
+    Attributes:
+        value: The JSON data. Values will automatically be converted to a JSON-compatible format.
+    """
+
+    value: Json = field(converter=lambda value: JsonArtifact.value_to_json(value), metadata={"serializable": True})
+
+    @classmethod
+    def value_to_json(cls, value: Any) -> Json:
+        if isinstance(value, str):
+            return json.loads(value)
+        else:
+            return json.loads(json.dumps(value))
```

Comment:
Why, dumps then loads? Deep copy?

Reply:
Need to convert to a JSON string before we can parse it as one.